internal/workload/podman/podman.go

```diff
@@ -316,3 +325,96 @@ func (p *podman) GenerateSystemdService(podId string, monitoringInterval uint) (
 
 	return svc, nil
 }
+
+// Retrieve all pods logs and send that to the given io.Writer
+func (p *podman) Logs(podID string, res io.Writer) (context.CancelFunc, error) {
+
+	podInfo, err := pods.Inspect(p.podmanConnection, podID, nil)
+	if err != nil {
+		return nil, err
+	}
+
+	ctx, cancel := context.WithCancel(p.podmanConnection)
+	writers := map[string]*bufio.Reader{}
+	for _, container := range podInfo.Containers {
+		res := &bytes.Buffer{}
```

Comment:
isn't there a risk of consuming too much memory for it? shouldn't it be dumped to a file?

Reply:
no, no a risk, another process will eat all and have an allowed buffer for it.